devops/scripts/benchmarks/benches/benchdnn.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (C) 2025 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
@pbalcer, shall "Part of the Unified-Runtime Project" part still be here for new files?

Reply:
yes, UR is still a standalone project.